src/form/Checkbox/CheckboxTheme.ts

```diff
@@ -51,11 +50,12 @@ const baseTheme: Partial<CheckboxTheme> = {
       large: 'text-lg'
     }
   },
-  check: '',
+  check: 'check',
   border: '',
   checkbox:
-    'flex items-center justify-center cursor-pointer focus-visible:outline-none',
-  disabled: 'opacity-60 cursor-not-allowed',
+    'checkbox flex items-center justify-center cursor-pointer focus-visible:outline-none',
+  checked: 'checked',
```

Comment:
looks like `checked` here isn't actually applying any styling, it's just used basically as state to conditionally apply styling to the checkbox below?

It kindaaa feels like maybe this doesn't belong in the theme file cuz it's sort of part of the checkbox logic. Could `checked` be it's own set of tw classes that get applied when the `checked` prop is true in `Checkbox`? Or is there a reason you can't do it that way

e.g. might be somethin like
```
checked: {
  base: `fill-blue-400`
  disabled: `fill-gray-400`
}
```

Reply:
what do the `check` and `checkbox` classes do?

Reply:
@ghsteff thanks for the comment, now I see what I did wrong, I do all the conditions in CheckboxTheme, although here I just need to describe the styles for all the options. I will try to move everything extra to the Checkbox component.